src/main/java/fi/aalto/cs/intellij/actions/EnablePluginsAction.java

```diff
@@ -0,0 +1,34 @@
+package fi.aalto.cs.intellij.actions;
+
+import com.intellij.ide.plugins.IdeaPluginDescriptor;
+import com.intellij.ide.plugins.PluginManager;
+import com.intellij.notification.Notification;
+import com.intellij.notification.NotificationAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import java.util.List;
+import java.util.Objects;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class EnablePluginsAction extends NotificationAction {
+
+  private List<IdeaPluginDescriptor> descriptors;
+
+  public EnablePluginsAction(
+      @Nls(capitalization = Nls.Capitalization.Title) @Nullable String text,
+      @NotNull List<IdeaPluginDescriptor> descriptors) {
+    super(text);
+    this.descriptors = descriptors;
+  }
+
+  /**
+   * Activate all the required plugins.
+   */
+  @Override
+  public void actionPerformed(@NotNull AnActionEvent e, @NotNull Notification notification) {
+    descriptors.forEach(descriptor -> Objects
+        .requireNonNull(PluginManager.getPlugin(descriptor.getPluginId())).setEnabled(true));
+    notification.expire();
+  }
```

Comment:
**actions:** so one approach I suggest is **not to use** anonymous actions, so to get them tested easier